gdal2mbtiles/gd_types.py

```diff
@@ -33,7 +33,7 @@
 
```

Comment:
I know that this isn't technically part of the 'API' for this project, but it might be worth mentioning that this file is renamed in the CHANGELOG, because this might affect users.

Reply:
Yes, agreed.  Good thing this PR also adds a CHANGELOG :)